docs/examples/tf/tf_wandb_hydra/README.md

```diff
@@ -0,0 +1,87 @@
+```
+
+## Run
+In order to run the example on SLURM you can just issue the following command from the example directory:
+```
+python train_mnist.py --multirun hydra/launcher=base +hours=1
+```
+
+### SLURM parametrization
+Different parameters can be set for the SLURM job, using the `hydra.launcher` config group.
+For example to launch a longer job, you can use:
+```
+python train_mnist.py --multirun hydra/launcher=base +hours=10 hydra.launcher.qos='qos_gpu-t3'
+```
+
+If you want to use more gpus:
+```
+python train_mnist.py --multirun hydra/launcher=base +hours=10 hydra.launcher.qos='qos_gpu-t3' hydra.launcher.gpus_per_node=4
+```
+
+### Weights&Biases
+`wandb` is run offline because the compute nodes are not connected to the internet.
+In order to have the results uploaded to the cloud, you need to manually sync them using the `wandb sync run_dir` command.
+The run directories are located in `$SCRATCH/wandb/jean-zay-doc`, but this can be changed using the `wandb.dir` config variable.
+You can also run a script to sync the runs before they are finished on a front node, for example using the script [here](https://gist.github.com/zaccharieramzi/3e1abc67aefac106ede2883c56ac8e1a).
+
+### Hydra and submitit outputs
+The outputs created by Hydra and submitit are located in the `multirun` directory.
+You can change this value by setting the `hydra.dir` config variable.
+
+### Batch jobs
+In order to batch multiple similar jobs you can use the sweep feature of Hydra.
+For example, if you want to run multiple training with different batch sizes, you can do the following:
+```
+python train_mnist.py --multirun hydra/launcher=base +hours=1 fit.batch_size=32,64,128
+```
+
+This can be extended to the grid search of a Cartesian product for example:
+```
+python train_mnist.py --multirun hydra/launcher=base +hours=1 fit.batch_size=32,64,128 compile.optimizer=rmsprop,adam
+```
+
+## Similar resources
+
+- [slurm-hydra-submitit](https://github.com/RaphaelMeudec/slurm-hydra-submitit) presents a similar concept in a more general case for any SLURM cluster, without W&B. In particular, it specifies [how to run specific parameters combinations grid search](https://github.com/RaphaelMeudec/slurm-hydra-submitit#specific-parameters-combinations).
+- [jz-hydra-submitit-launcher](https://github.com/zaccharieramzi/jz-hydra-submitit-launcher) a pip installable (`pip install jz-hydra-submitit-launcher`) custom launcher that has the correct default for JZ, and several default configurations:
+```
+hydra-submitit-launch train_mnist.py dev hydra.launcher.setup=\["'#SBATCH -C v100-32g'","'export WANDB_MODE=offline'"\]
+```
+
+
+## References
+- Weights&Biases: https://wandb.ai/site
+- Hydra: https://hydra.cc/
+- Submitit: https://github.com/facebookincubator/submitit
+- Hydra submitit launcher: https://hydra.cc/docs/plugins/submitit_launcher/
+
+## Alternatives
+
+To Weights&Biases:
+- MLFlow
+- Tensorboard
+
+To Hydra:
+- argparse
+- click
```

Comment:
I would say that Dask is really different than Hydra: it does neither the configuration nor the command line interface creation.
Indeed it allows to launch many things in parallel, but I think it would qualify more as a Hydra launcher than anything.